vaadin-components-gwt/src/main/java/com/vaadin/components/grid/GridComponent.java

```diff
@@ -439,7 +443,7 @@ public void sort(SortEvent<Object> event) {
             jsSort.push(sortOrder);
         }
         $(container).trigger("sort");
-        refresh();
+        refresh(false);
```

Comment:
Well if that's true, then we can revert all the changes related with `refresh()`. How certain are you about this? :smiley: